### PR TITLE
set up project skeleton

### PR DESCRIPTION
**Pull request by miwst**
I spent about an hour on this assignment.

I developed a sense of how the portfolio will be laid out.

I've decided on how to use a constructor function to display my projects on later implementations.

### CODE REVIEW

File: scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+$(() => {
+
+  const projects = [
```
Comment (Tenlia) [inline scripts/script.js]: nicely set-up

File: scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+$(() => {
```
Comment (Tenlia) [inline scripts/script.js]: What is this doing?

File: scripts/script.js (hunk 1 above)
Comment (Tenlia) [inline scripts/script.js]: your objects should be coming from a different document or external source.

Comment (Tenlia) [inline scripts/script.js]: You don't need to set that up, yet. Unless you wanted to show a demonstration.

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (Tenlia) [inline styles/style.css]: nice, simplistic. I like simple styles

File: scripts/script.js (hunk 2 above)
Comment (miwst) [inline scripts/script.js]: I was just putting it in strict mode and doing a jQuery .ready function

File: scripts/script.js (hunk 1 above)
Comment (miwst) [inline scripts/script.js]: I was planning on having an array with the project tiles and their associated hrefs to loop through.  Should that array be located in another file?

File: styles/style.css (hunk 3 above)
Comment (miwst) [inline styles/style.css]: I just had that in there initially to make sure my css was linked to the html.  oops.

File: scripts/script.js (hunk 2 above)
Comment (Tenlia) [inline scripts/script.js]: Okay, I'd never seen it done that way before. Nice

**Pull request merged**